scripts/azureml-assets/azureml/assets/environment/build.py

```diff
@@ -62,6 +62,12 @@ def create_acr_task(image_name: str,
             'id': 'test',
             'cmd': f"$Registry/{image_name} {test_command}"
         })
+        task['steps'].append({
+            'id': 'scan',
+            'cmd': f"$Registry/{image_name} wget https://github.com/aquasecurity/trivy/releases/download/v0.42.1/trivy_0.42.1_Linux-64bit.deb && \
+                    dpkg -i trivy_0.42.1_Linux-64bit.deb && trivy fs --scanners vuln /",
+            'ignoreErrors': True

```

Comment:
While this mostly works, it introduces a bunch of whitespace into the string. What do you think of this?
```suggestion
            'cmd': (
                       f"$Registry/{image_name} "
                       "wget https://github.com/aquasecurity/trivy/releases/download/v0.42.1/trivy_0.42.1_Linux-64bit.deb && "
                       "dpkg -i trivy_0.42.1_Linux-64bit.deb && "
                       "trivy fs --scanners vuln /"
                   ),
            'ignoreErrors': True
```

Reply:
That sounds better, thank you!